scripts/unsupported/revert_sam/ExampleBamToUnmappedBams.json

```diff
@@ -0,0 +1,7 @@
+{
+  "CramToUnmappedBams.input_bam": "gs://input_file.bam",
```

Comment:
should the values for these be `"String"` or `Int` to make sure people understand this wont actually run

Reply:
Sure.